www/unstable/userguide/weekyears.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: userguide
+title: Week years
+category: advanced
+weight: 3037
+---
+
+Introducing week-years
+===
+
+Most of the time, dates are represented as a year, month and day in
+a particular calendar system - occasionally with an era as well. An
+alternative approach is to represent a date as a week-year, week,
+and day-of-week. This is performed relative to a specific calendar
+system, along with a rule used to work out when each week-year starts.
+
+For example, using the ISO-8601 rule, the year/month/day combination
+"2016, February, 4" is equivalently week-year 2016, week 5,
+Thursday.
+
+The two representations are exactly equivalent: every day
+can be represented in either form. Week-years are more widely used
+in some countries than in others, typically in business or
+educational settings.
+
+Noda Time features for week-years
+===
+
+As of Noda Time 2.0, week-year rules are entirely separate from calendar
+systems. A rule is represented by the
+[`IWeekYearRule`](noda-type://NodaTime.Calendars.IWeekYearRule)
+interface, and built-in implementations are obtained via factory
+members in the
+[`WeekYearRules`](noda-type://NodaTime.Calendars.WeekYearRules)
+static class.
+
+Although the `GetWeeksInWeekYear` and `GetLocalDate` methods accept
+`CalendarSystem` parameters, extension methods in
+[`WeekYearRuleExtensions`](noda-type://NodaTime.Calendars.WeekYearRuleExtensions)
+are equivalent but without the parameters, simply delegating to the
```

Comment:
Although the `GetWeeksInWeekYear` and `GetLocalDate` methods require a `CalendarSystem`, we also provide equivalent extension methods in `WeekYearRuleExtensions` that the default to the ISO calendar system, delegating to the existing `IWeekYearRule` methods. This reduces...